nuitka/build/static_src/CompiledFunctionType.c

```diff
@@ -570,7 +602,12 @@ PyTypeObject Nuitka_Function_Type = {
 #endif
 };
 
-void _initCompiledFunctionType(void) { PyType_Ready(&Nuitka_Function_Type); }
+void _initCompiledFunctionType(void) {
+    // PyModule_AddObject(Nuitka_Function_Type.m_module, "Meta", (PyObject *) &MetaType);
+     Nuitka_Function_Type.ob_type = &MetaType;
+    //  Py_TYPE(&Nuitka_Function_Type) = &MetaType;
```

Comment:
The uncommented version is more explicit. The commented one can be removed I guess.

Reply:
@kayhayen but that doesn't works. It gives error `error: ‘PyTypeObject {aka struct _typeobject}’ has no member named ‘ob_type’`.

Reply:
Check out this: https://docs.python.org/3/c-api/typeobj.html